docs/user-management/manage-permissions.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 4
+title: Manage Permissions
+---
+
+This section explains user permissions and how to change them for users and API keys.
+
+## Permissions
+
+Each GeoFabric has the following global permission levels for users and API keys:
+
+* **Administrate -** Full read and write access to all files in the GeoFabric.
+* **Access -** Read access to all files in the GeoFabric. No write access.
+* **No Access -** No access to any files in the GeoFabric.
+* **Use Default -** Uses default settings as specified on the bottom row.
+
+Click the name of the GeoFabric to expand it and view permissions for specific features. Each GDN feature has the following permission levels:
+
+* **Read/Write -** Full access to the feature and related files.
+* **Read Only -** Read access to the feature and related files. No write access.
+* **No Access -** No access to the feature and related files.
+* **Use Default -** Uses default settings as specified on the bottom row.
+
+## Change Permissions
+
+You can change permissions for users and API keys.
```

Comment:
I think we need to explain this a little more, make clear all the things you can give access to. It might actually require separate tasks.

Change Permissions for GeoFabric
Change Billing Permissions
Change Collection Permissions